### ISSUE TITLE
Configurable health check warnings

### ISSUE THREAD

**Issue by Tiwas** [Radarr/Radarr#10542]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

No, except for constant warnings by email.

### Describe the solution you'd like

It would be nice to have some control over which warnings you want, or even better - to be able to set some sort of timeout.

On my system, having indexers go down for a little while is not unusual. I don't need to know about it, at least not until it's been a problem for X minutes/hours/days.

I guess the easiest would be to just turn warnings on or off, or be able to select which ones you want emails about.

### Describe alternatives you've considered

As getting rid of Radarr isn't an option, my alternatives are ignoring the emails or creating a rule in my mail client.

### Anything else?

.

**Comment by bakerboy448** [comment]
RELATED TO https://github.com/Radarr/Radarr/issues/7570
